doc/local.conf

```diff
@@ -254,6 +254,9 @@ WKS_FILE = "aos-minimal.wks.in"
 # Disable meta-virtualization warnings
 SKIP_META_VIRT_SANITY_CHECK = "1"
 
+# Define which runner will be used. Default runc
```

Comment:
Please add: "which runner will be used to run AoS services" and also specify other possible options. For example: "currently supported runc and crun. Default is runc."